pootle/apps/pootle_misc/templatetags/search.py

```diff
@@ -17,6 +17,16 @@
 
 @register.inclusion_tag('core/search.html', takes_context=True)
 def render_search(context):
+    search_form = make_search_form(request=context['request'])
+    if context["page"] != "translate" and not context["can_translate_stats"]:
+        search_form.fields["search"].widget.attrs["readonly"] = "readonly"
+        search_form.fields["search"].widget.attrs["disabled"] = True
+        search_form.fields["search"].widget.attrs["title"] = ""
+        search_form.fields["search"].widget.attrs[
+            "placeholder"] = "Search unavailable"
+        search_form.fields["search"].widget.attrs[
+            "style"] = "background: #eee; font-style; italics"
```

Comment:
IMHO we should set some CSS class and keep CSS styling in stylesheets, instead of here.